pkg/writer/encoding.go

```diff
@@ -136,18 +136,6 @@ func newCollection(apiOp *types.APIRequest, list types.APIObjectList) *types.Gen
 		},
 	}
 
-	partial := list.Continue != "" || apiOp.Query.Get("continue") != ""
```

Comment:
Is the idea that this code is safe to remove since we never set the "Continue" value in our response back to the user, so the user should never make a request with a continue value and we should never construct the pagination option, thereby eliminating the only use of the pagination struct?

Reply:
Not exactly, "continue" does get set if the result from k8s is partial, see https://kubernetes.io/docs/reference/using-api/api-concepts/#retrieving-large-results-sets-in-chunks and https://github.com/rancher/steve/blob/a10fe811f58ff7d92ffee51dd3f11010afbaf115/pkg/stores/partition/store.go#L111

The idea was that steve doesn't use this Pagination object, and including it in the list result is confusing and unnecessary. The only information the client really needs is the continue token.

It's been a minute since I deleted this and your question made me look closer at why I chose to remove it and I found that the dashboard is actually using this object, especially the "partial" and "next" fields. I'll put it back, and we'll have to document that there's a difference between `Pagination` for server-side chunking and `Pages` for rancher-side pagination.